ibft/ibft_decided_test.go

```diff
@@ -270,7 +271,7 @@ func TestSyncAfterDecided(t *testing.T) {
 		Round:       3,
 		SeqNumber:   10,
 		ValidatorPk: validatorPK(sks).Serialize(),
-		Lambda:      []byte("Lambda"),
+		Lambda:      []byte(IdentifierFormat(validatorPK(sks).Serialize(), beacon.RoleAttester)),
```

Comment:
@nivBlox test is generic.. it doesn't matter what the lambda is.. let's change it back